Formula/crystal.rb

```diff
@@ -60,6 +60,7 @@ def install
     ENV.append_path "PATH", "boot/bin"
     ENV.append_path "CRYSTAL_LIBRARY_PATH", Formula["bdw-gc"].lib
     ENV.append_path "CRYSTAL_LIBRARY_PATH", ENV["HOMEBREW_LIBRARY_PATHS"]
+    ENV.append_path "LLVM_CONFIG", Formula["llvm"].bin/"llvm-config"
```

Comment:
```suggestion
    ENV.append_path "CRYSTAL_LIBRARY_PATH", Formula["bdw-gc"].opt_lib
    ENV.append_path "CRYSTAL_LIBRARY_PATH", ENV["HOMEBREW_LIBRARY_PATHS"]
    ENV.append_path "LLVM_CONFIG", Formula["llvm"].opt_bin/"llvm-config"
```
Let's make these refer to paths that don't change between version bumps.

Reply:
Oh, and please increment `revision` from `2` to `3`, so that users who have this installed get the updated version of the formula when they do `brew upgrade`.